.github/workflows/build_all.yml

```diff
@@ -63,10 +63,16 @@ jobs:
       - name: Fetch tags and unshallow
         run: git fetch --unshallow --tags
 
+      - run: brew update
```

Comment:
A lot of work :)

What's wrong with the stock Python in the runner?

Reply:
I do not trust Github at all, there was a cascading laundry list of problems with it and I *do* trust brew.

But mostly, the issue is that github broke brew itself, and any formula that attempts to install python, either directly or as a dependency, will error out -- and that includes any wrapdb package that specifies `brew_packages` if that brew package pulls in python anywhere in the dependency tree.

So we must unbreak brew in order to guarantee that arbitrary `brew install XXX` will "just work" -- and that means nuking the github actions one from high orbit until no traces remain.

Reply:
The CI also supports vcpkg. Wonder which is better...

Reply:
I don't know how true that indirect python comment is. For example, Orcus specifies boost, which definitely pulls python. CI didn't fail though.

Reply:
Which CI is that? I didn't think we recently changed orcus.

Reply:
I ran manual build all runs on here after my changes.

Reply:
I went and looked up boost in homebrew, it doesn't depend on python at all. boost_python3 does, but that's a separate formula. (It will also install `python@3.11`, which isn't the same as the `python` alias, but does conflict once again with Github Actions.)

Reply:
Will look when CI is done.

Reply:
Hmm seems correct.